cmd/remove.go

```diff
@@ -130,3 +139,31 @@ func clusterRemoveLocal(ctx *cli.Context) error {
 
 	return ClusterRemove(context.Background(), rkeConfig, nil, nil, true, "")
 }
+
+func clusterRemoveDind(ctx *cli.Context) error {
+	clusterFile, filePath, err := resolveClusterFile(ctx)
+	if err != nil {
+		return fmt.Errorf("Failed to resolve cluster file: %v", err)
+	}
+
+	rkeConfig, err := cluster.ParseConfig(clusterFile)
+	if err != nil {
+		return fmt.Errorf("Failed to parse cluster file: %v", err)
+	}
+
+	rkeConfig, err = setOptionsFromCLI(ctx, rkeConfig)
+	if err != nil {
+		return err
+	}
+
+	for _, node := range rkeConfig.Nodes {
+		if err = dind.RmoveDindContainer(context.Background(), node.Address); err != nil {
+			return err
+		}
+		time.Sleep(DINDWaitTime * time.Second)
```

Comment:
why do we need to sleep here? should we validate smth after sleep is done?

Reply:
oh that's an error the sleep was supposed to be after creation to give time to the container to be ready, will fix